dev/rclone.ts

```diff
@@ -0,0 +1,697 @@
+// TODO: dynamically suggest paths
+const remote: Fig.Arg = {
+  name: "remote:",
+  generators: [
+    {
+      script: "rclone listremotes",
+      postProcess: (list) =>
+        list.split("\n").map((remote) => ({ name: remote })),
+    },
+  ],
+};
+const hashes = [
+  "MD5",
+  "SHA-1",
+  "DropboxHash",
+  "QuickXorHash",
+  "Whirlpool",
+  "CRC-32",
+  "MailruHash",
+];
+const listOptions = [
```

Comment:
Could you please add 
```suggestion
const listOptions: Fig.Option[] = [
```
here :)